util/expensivequery/expensivequery.go

```diff
@@ -70,10 +70,22 @@ func (eqh *Handle) Run() {
 					logExpensiveQuery(costTime, info)
 					info.ExceedExpensiveTimeThresh = true
 				}
-
 				if info.MaxExecutionTime > 0 && costTime > time.Duration(info.MaxExecutionTime)*time.Millisecond {
+					logutil.BgLogger().Warn("execution timeout, kill it", zap.Duration("costTime", costTime),
+						zap.Duration("maxExecutionTime", time.Duration(info.MaxExecutionTime)*time.Millisecond), zap.String("processInfo", info.String()))
 					sm.Kill(info.ID, true)
 				}
+				if strings.Contains(info.String(), "analyze") {
+					logutil.BgLogger().Info("find analyze")
```

Comment:
Why this log?

Reply:
I forget to remove the debug code. Updated.